training/templates/base.html

```diff
@@ -1,40 +1,128 @@
 {% load static %}
+
 <!DOCTYPE HTML>
-<html>
+<html lang="en">
 <head>
-
 	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 	<meta name="viewport" content="maximum-scale=1.0">
 	<meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
-	<title>Galaxy TIaaS {% block title %}{% endblock %}</title>
-	<link href="{{ settings.TIAAS_DOMAIN }}/static/style/base.css" media="screen" rel="stylesheet" type="text/css" />
-	<link href="{{ settings.TIAAS_DOMAIN }}/static/dist/base.css" media="screen" rel="stylesheet" type="text/css" />
-
-	<style type="text/css" media="all">
-		#center{ padding-right: 10em; padding-left: 10em; overflow-y: auto; }
-	</style>
-	 <link  href="{% static 'training/style/tiaas.css' %}" media="screen" rel="stylesheet" type="text/css" />
+	<title>{{ settings.TIAAS_OWNER }} TIaaS {% block title %}{% endblock %}</title>
+
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+	<link rel="stylesheet" href="{% static 'vendor/fontawesome-5.15.4/css/all.min.css' %}"/>
+	<link rel="stylesheet" href="{% static 'vendor/material-icons.css' %}">
+
+	<!-- Use local Galaxy's stylesheet -->
+	<link rel="stylesheet" href="/static/dist/base.css">
```

Comment:
(reproduction of comment on another file.)

This may not always be correct, that was the reason for using `{{ TIAAS_DOMAN }}`, if they've prefixed their galaxy with a proxy prefix, e.g. `/galaxy/` then this breaks.

Reply:
Thanks, fixed now. I've also refactored the styling a little since I realised how this is working. I tested it with stylesheets from main, EU and AU and they seem to cascade nicely now. e.g:

![image](https://user-images.githubusercontent.com/42562517/192426073-619f2a2d-07ce-47d3-bd66-f13da5936b3b.png)

Reply:
Looks great!